recipes/py-markdown-table/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "py-markdown-table" %}
+{% set version = "1.3.0" %}
+{% set python_min = "3.8" %}
```

Comment:
Conda-forge doesn't support EoL Python. If you really need this we can keep, but we won't fix any possible issues that are due to older Pythons.
  
```suggestion
```